terminal-dashboard/table.bash

```diff
@@ -59,6 +69,8 @@ for LIB in $(get_libraries_by_collection "${COLLECTION}" ); do
 
         if [[ $ARCH == "i386" && $VER == "focal" ]]; then
```

Comment:
There won't be more i386 Ubuntu support by now so we can turn this into: 
```suggestion
        if [[ $ARCH == "i386" && $VER != "bionic" ]]; then
```
and remove the new `elif` for jammy.

Reply:
This applies for Debian too, so we need to account for `buster` somewhere